zillow_crawler/main.py

```diff
@@ -68,7 +83,8 @@ def main():
         # ]
         # live.update(Panel(Columns(renderables, equal=True), title="Books"))
         session.close()
-
+        print("\nTotal crawl time: "+str(f'{total_crawl_time:0.2f} sec'))
+        print("Total houses scraped: "+str(total_houses))
```

Comment:
I suggest adding the total pages scraped.

Reply:
done